types/api.ts

```diff
@@ -0,0 +1,10 @@
+export interface ApiChange {
+  version: string | string[];
```

Comment:
Is it possible to have just `string[]` as the type?

Reply:
we could do, but I am not sure about it. I need someone to decide it. Thank you.